components/CountyGuidelines.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="CountyGuidelines">
+    <div v-for="(item, i) in info[county].info" :key="i">
+      <v-card class="guidelines">
+        <p class="Header">
+          {{ item.header }}
+        </p>
+        <!-- <p class="time-updated">
+          Last updated dd/mm/yy
+        </p> -->
+        <div v-for="(body, j) in item.body" :key="j">
```

Comment:
For key I prob would use a description in combination with just the index. Keys within a v-for must be unique. So your nested child keys will prob get dups cuz it is nested in this v-for of the body. A way to solve this would be to add a description to the key `body-${j}` and converting this to key="body-0".

[does v-for key need to be unique across the entire component](https://forum.vuejs.org/t/v-for-key-does-the-key-need-to-be-unique-across-the-entire-component/28040)

Reply:
@thuongho you mind clarifying this a little more like maybe a few more examples? Im following what you are saying but I still can't quite grasp it.

Reply:
sure @devdav1999. Since body is your most outer loop, you can do 
```
:key=`body-${j}`
```
then for your inner loops like the next one is content, you can do 
```
:key=`content-${k}`
```
and so forth...

Reply:
Ok I applied this @thuongho. If you don't mind checking it when you have time to see if I did it right? Thanks again